src/types/Health.ts

```diff
@@ -202,49 +245,64 @@ interface HealthContext {
 }
 
 export class ServiceHealth extends Health {
-  public static fromJson = (json: any, ctx: HealthContext) => new ServiceHealth(json.requests, ctx);
+  public static fromJson = (ns: string, srv: string, json: any, ctx: HealthContext) =>
+    new ServiceHealth(ns, srv, json.requests, ctx);
 
-  private static computeItems(requests: RequestHealth, ctx: HealthContext): HealthItem[] {
+  private static computeItems(ns: string, srv: string, requests: RequestHealth, ctx: HealthContext): HealthConfig {
     const items: HealthItem[] = [];
+    let statusConfig: HealthItemConfig | undefined = undefined;
     if (ctx.hasSidecar) {
       // Request errors
-      const reqErrorsRatio = getRequestErrorsStatus(requests.errorRatio);
-      const reqErrorsText = reqErrorsRatio.status === NA ? 'No requests' : reqErrorsRatio.value.toFixed(2) + '%';
+      const reqError = calculateErrorRate(ns, srv, 'service', requests);
+      const reqErrorsText =
+        reqError.errorRatio.global.status.status === NA
+          ? 'No requests'
+          : reqError.errorRatio.global.status.value.toFixed(2) + '%';
       const item: HealthItem = {
         title: 'Error Rate over ' + getName(ctx.rateInterval).toLowerCase(),
```

Comment:
I don't really like `Error Rate over 1hr` for this information.  I'd prefer to see, `Traffic Status (Last 1hr)`.    This is more consistient with `Pod Status` above.  Also `Last 1hr` is consistent with the duration drop down.  And for me, I don't like seeing the word 'Error' when things are healthy.

![image](https://user-images.githubusercontent.com/2104052/92161121-25361980-edfe-11ea-8e83-404c7a30cb0e.png)

One more note, I'm not sure it's intentional but the icon used to have a text component, it no longer does.  In the past:

![image](https://user-images.githubusercontent.com/2104052/92161736-008e7180-edff-11ea-9092-7e9cd1d080e5.png)

Actually, one last note :-)  I think maybe HealthItem should have a static name.  I see above in the code this dangerous test, which will break on a change to the item title:

`  return item.title.startsWith('Error Rate') ? (`